src/Server_TEST.cc

```diff
@@ -849,7 +849,7 @@ TEST_P(ServerFixture, Seed)
 /////////////////////////////////////////////////
```

Comment:
There are more instances in this file:

```
$ grep -n "IGN_GAZEBO_RESOURCE_PATH" src/Server_TEST.cc 
1039:  common::setenv("IGN_GAZEBO_RESOURCE_PATH", "");
1074:  common::setenv("IGN_GAZEBO_RESOURCE_PATH", "");
1087:  common::setenv("IGN_GAZEBO_RESOURCE_PATH", PROJECT_SOURCE_PATH);
1099:  common::setenv("IGN_GAZEBO_RESOURCE_PATH",
```

Reply:
https://github.com/gazebosim/gz-sim/pull/1518/commits/35d865fb2f5cb06269dce70704c675cea74902c7